### ISSUE TITLE
Focus Visible - Accessibility Issue

### ISSUE THREAD

**Issue by phucduong86** [CodingTrain/website#2050]
There seems to be an issue with the dropdown button implementation on the nav bar.
In Mobile mode those button was not properly removed and can still be focused off screen and not visible to keyboard user.
This only happens on first page load, after that it is fine but it is a Critical WCAG issue.

![image](https://user-images.githubusercontent.com/21692768/72214026-ba7b4b00-34c7-11ea-87e2-37ecec8715fa.png)

**Comment by violetcraze** [comment]
@phucduong86 Thanks for catching this problem. I'll work on it this weekend. It looks like you're using a tool would love to know what it is so I could run it to make sure the problem gets fixed.

**Comment by shiffman** [comment]
I'm not sure if this is what @phucduong86 uses, but the one I typically check the site with is:

https://wave.webaim.org/extension/

**Comment by phucduong86** [comment]
@violetcraze @shiffman 
WAVE and Axe extensions are both great automated testing tools.

For this issue specifically (and for keyboard focus visible/keyboard trap issue) it would require manual testing. If you tab through the page and you don't know where the focus is at then that is an accessibility issue.
When you use the screen reader (NVDA on Windows or VoiceOver in Mac which are both free), it will announce the element that is focused on that's how you identify the problem elements.
